aviary/validation_cases/benchmark_tests/test_subsystems_within_a_mission.py

```diff
@@ -15,48 +14,36 @@
 @use_tempdirs
 class TestSubsystemsMission(unittest.TestCase):
     def setUp(self):
-        polar_file = "subsystems/aerodynamics/gasp_based/data/large_single_aisle_1_aero_free.txt"
-        aero_data = pkg_resources.resource_filename("aviary", polar_file)
-
         self.phase_info = {
             'pre_mission': {
                 'include_takeoff': False,
                 'external_subsystems': [ArrayGuessSubsystemBuilder(), AdditionalArrayGuessSubsystemBuilder()],
-                'optimize_mass': False,
+                'optimize_mass': True,
             },
             'cruise': {
-                'subsystem_options': {
-                    'core_aerodynamics': {'method': 'solved_alpha', 'aero_data': aero_data, 'training_data': False}
-                },
+                "subsystem_options": {"core_aerodynamics": {"method": "computed"}},
```

Comment:
Why was aero method changed? It should be mission agnostic (and if it is somehow not, then we need a bug report!)

Reply:
Changed for no particular reason; I copied-pasted this phase_info from another place. It is indeed mission agnostic